lib/geoengineer/resources/aws/api_gateway/helpers.rb

```diff
@@ -166,5 +166,24 @@ def _remote_rest_api_gateway_responses(provider)
         end
       end
     end
+
+    def _fetch_remote_rest_api_authorizers(provider, rest_api)
+      resources = _client(provider).get_authorizers(
+        { rest_api_id: rest_api[:_terraform_id] }
+      )['items']
+      resources.map(&:to_h).map do |ga|
+        ga[:_terraform_id] = ga[:id]
+        ga[:_geo_id]       = "#{rest_api[:_geo_id]}::#{ga[:name]}"
```

Comment:
I am not sure if this is the right way to derive the geo_id, but it looks to match what is done from other resources.

Reply:
geo_id is up to you! As long as provides uniqueness per resource - it is used as a primary key for identifying resources. This seems like this should be unique.